frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebar.scss

```
@@ -1,4 +1,11 @@
 .opp-task-sidebar {
+  width: 550px;
```

Just a small improvement, we used this CSS in the old Sidebar.scss:

```suggestion
$overview-sidebar-width: 550px;

.opp-task-sidebar {
  max-width: $overview-sidebar-width;
```

---

updated.